web/public/themes/nex-shell.css

```
@@ -0,0 +1,265 @@
+   WUPHF Theme: Shell
+   The operator shell's dark technical manual, applied to the office.
+   Paper-dark surfaces, soft off-white ink, cobalt figure accents,
+   speckle + construction-grid atmosphere. No brand purple: color is
+   reserved for meaning (green running, amber warning, red error) and
+   cobalt marks links, selection, and focus. Values mirror
+   web/src/styles/operator-shell.css `.opr-root` — move them together.
+   Structural rules live in nex.css (scoped [data-theme^="nex"]) and
+   are inherited here; this file only overrides the semantic layer.
+   ═══════════════════════════════════════════════════════════════ */
+
+@import url("./nex.css");
```

Stylelint (17.14.0)

[error] Expected "url("./nex.css")" to be ""./nex.css"" (import-notation)

(import-notation)